src/training/unicharset/lstmtester.cpp

```diff
@@ -118,8 +118,8 @@ std::string LSTMTester::RunEvalSync(int iteration, const double *training_errors
   std::string result;
```

Comment:
There's one more: one line 109 (`Line Char error rate` → `Line BCER`) – cannot use GH suggestions.

Reply:
That's fixed now, too.